### PR TITLE
remove URL tag in ingress span

### PR DESCRIPTION
**Pull request by szuecs**
removing URL tag in ingress span can save a lot of bytes for example with elastic search cases
Interesting observation: proxy span has no kv pairs from query, that's why I let it there.

Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

File: proxy/proxy.go [hunk 1]
```diff
@@ -1449,7 +1449,6 @@ func (p *Proxy) Close() error {
 func (p *Proxy) setCommonSpanInfo(u *url.URL, r *http.Request, s ot.Span) {
 	p.tracing.
 		setTag(s, ComponentTag, "skipper").
-		setTag(s, HTTPUrlTag, u.String()).
```
Comment (pitr) [inline proxy/proxy.go:1452]: you can replace with log instead

Comment (lmineiro) [reply proxy/proxy.go:1452]: Replacing the tag with a log is not going to help reduce the size of the gRPC message

**Comment by aryszka** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**